pkgs/applications/window-managers/notion/default.nix

```diff
@@ -1,45 +1,47 @@
-{
-  enableXft ? true, libXft ? null, patches ? [], stdenv, lua, gettext, pkgconfig, xlibsWrapper, libXinerama, libXrandr, libX11,
-  xterm, xmessage, makeWrapper, fetchFromGitHub, mandoc, which
+{ stdenv, fetchFromGitHub, pkgconfig
+, lua, gettext, which, groff
+, xmessage, xterm
+, readline, fontconfig
+, xlibsWrapper
+, libX11, libXext, libSM
+, libXinerama, libXrandr
+, libXft
+, makeWrapper
 }:
 
-assert enableXft -> libXft != null;
-
-let
+stdenv.mkDerivation rec{
   pname = "notion";
-  version = "3-2017050501";
-  inherit patches;
-in
-stdenv.mkDerivation {
-  name = "${pname}-${version}";
-  meta = with stdenv.lib; {
-    description = "Tiling tabbed window manager, follow-on to the ion window manager";
-    homepage = http://notion.sourceforge.net;
-    platforms = platforms.linux;
-    license   = licenses.notion_lgpl;
-    maintainers = with maintainers; [jfb];
-  };
+  version = "2020-01-07";
```

Comment:
It seems kind of weird to update this package from a released upstream tag to a timestamped git snapshot of the next (as of yet unreleased) major version introducing significant changes (https://notionwm.net/migration.html).

Thanks for not using the same version format, that does make it less confusing.

I'm not opposed to it, but my impression was that Nix prefers packaging released tags.

Reply:
Indeed, this package should be named “notion-unstable” and the version should be set to the commit date.

Reply:
> I'm not opposed to it, but my impression was that Nix prefers packaging released tags.

Initially I think "well, maybe the master branch fixed the compilation issues", then I tried it before anything else.